packages/react-composites/tests/browser/chat/app/index.tsx

```diff
@@ -93,3 +99,23 @@ function App(): JSX.Element {
 }
 
 ReactDOM.render(<App />, document.getElementById('root'));
+
+function getMessageContentInUppercase(messageProps: MessageProps): string {
+  if (
```

Comment:
This is a tagged union:

```suggestion
  const message = messageProps.message;
  switch (message.messageType) {
    case 'chat':
    case 'custom':
    case 'content':
      return message.content.toUpperCase();
    case 'system':
      switch(message.systemMessageType) {
        case 'topicUpdated':
          return message.topic.toUpperCase();
        case 'participantAdded':
          return `Participants Added: ${message.participants.map((p) => p.displayName).join(',')}`.toUpperCase();
        case 'participantRemoved':
          return `Participants Removed: ${message.participants.map((p) => p.displayName).join(',')}`.toUpperCase();
        default:
          return 'CUSTOM SYSTEM MESSAGE';
      }
      default:
        return 'CUSTOM MESSAGE';
  }
```

Reply:
Thanks, updated